python/GafferDispatchTest/LocalDispatcherTest.py

```diff
@@ -492,7 +492,11 @@ def testFailure( self ) :
 		s["n2"]["preTasks"][0].setInput( s["n3"]["task"] )
 
 		dispatcher = GafferDispatch.Dispatcher.create( "LocalTest" )
-		dispatcher.dispatch( [ s["n1"] ] )
+		
+		try :
+			dispatcher.dispatch( [ s["n1"] ] )
+		except Exception as e :
+			self.assertTrue( "No such file or directory" in str(e) )
```

Comment:
Nitpick : this is what `assertRaisesRegexp()` is for...

Reply:
Addressed in 5d9a43d2c2bb335682bf02ba14509b6c20baacc2